File: Julia/8kyu/8kyu-multiply.jl
# Multiply
# Language: Julia
# Version: 1.5
# Difficulty: 8 kyu

module Solution
  export multiply
  function multiply(a, b)
    a * b
  end
end


# Tests
using FactCheck
using .Solution

facts("multiply") do
  context("fixed tests") do
    @fact multiply(2, 2) --> 4
    @fact multiply(1, 1) --> 1
    @fact multiply(3, 2) --> 6
  end
end

File: JavaScript/8kyu/8kyu-multiply.js
// Multiply
// Language: JavaScript
// Version: Node v18.x
// Difficulty: 8 kyu

function multiply(a, b){
  return a * b;
}


// Tests

const assert = require("chai").assert;

describe("Multiply", () => {
  it("fixed tests", () => {
    assert.strictEqual(multiply(1,1), 1);
    assert.strictEqual(multiply(2,1), 2);
    assert.strictEqual(multiply(2,2), 4);
    assert.strictEqual(multiply(3,5), 15);   
  });
});